src/docfx/lib/HashUtility.cs

```diff
@@ -54,6 +55,16 @@ public static string GetFileSha1Hash(string fileName)
             }
         }
 
+        public static string GetSortedMonikerListHash(List<string> monikers)
+        {
+            if (monikers == null || monikers.Count == 0)
```

Comment:
Why do we want to put a `moninker` specific method in `HashUtility`? And do we really need another method for the null check?